App.js

```diff
@@ -1,19 +1,42 @@
-import React from 'react';
+import React, {useContext} from 'react';
 import {createStackNavigator} from '@react-navigation/stack';
-import NavigationContainer from '@react-navigation/native/src/NavigationContainer';
 import {Home, Login} from './src/views';
+import AuthContext from './src/store/authContext';
+import useGlobalState from './src/store/store';
+import {NavigationContainer} from '@react-navigation/native';
 
 const Stack = createStackNavigator();
 
+const AppWrapper = () => {
+  const store = useGlobalState();
+  return (
+    <AuthContext.Provider value={store}>
+      <App />
+    </AuthContext.Provider>
+  );
+};
+
 const App = () => {
+  const {isLoggedIn} = useContext(AuthContext);
+
   return (
     <NavigationContainer>
-      <Stack.Navigator initialRouteName="Login">
-        <Stack.Screen name="Login" component={Login} />
-        <Stack.Screen name="Home" component={Home} />
+      <Stack.Navigator>
+        {isLoggedIn() === false ? (
```

Comment:
Se usares o valor do **loggedIn** aqui não precisas de chamar a função.
E não uses if's negativos quando podes trocar a ordem porque pode ficar confuso de ler.
```suggestion
        {isLoggedIn() === false ? (
```